pilopt/src/lib.rs

```diff
@@ -104,10 +56,10 @@ fn remove_constant_fixed_columns<T: FieldElement>(pil_file: &mut Analyzed<T>) {
             name: _,
             index,
             next: _,
-            poly_id,
+            poly_id: Some(poly_id),
```

Comment:
This is needed because we can still have constant references inside the fixed column code.